readme.md

```diff
@@ -14,11 +14,9 @@ Install with [npm](https://npmjs.com):
 npm install apollo-upload-client
 ```
 
-[Apollo Boost](https://npm.im/apollo-boost) doesn’t allow link customization; if you are using it [migrate to a manual Apollo Client setup](https://apollographql.com/docs/react/migrating/boost-migration).
+[Apollo Client](https://apollographql.com/docs/react) can only have 1 [terminating Apollo Link](https://apollographql.com/docs/link/overview/#terminating-links) that sends the GraphQL requests; if one such as [`HttpLink`](https://apollographql.com/docs/link/links/http) is already setup, remove it. Instead of adding the options `uri`, `headers` and `credentials` in the `ApolloClient` constructor, do so in [`createUploadLink`](#function-createuploadlink).
```

Comment:
@hoangvvo I'm not so sure that setting `uri`, `headers` and `credentials` in the `ApolloClient` constructor won't work. Maybe it does?

Reply:
It doesn't. It's for the default `HttpLink` https://github.com/apollographql/apollo-client/blob/master/src/ApolloClient.ts#L135 if no `link` is defined in the constructor. This is to avoid confusion.

Reply:
Ok I just tried it, and it doesn't work. I wonder if we can get it to though.

Reply:
I look through and don't think that `options.uri` is passed down to any link. I guess it's just like a `ApolloBoost` thingy.

Reply:
I was thinking it might set it in the link context or something. It looks like the `HttpLink` isn't able to figure out the `uri` set in the constructor options, so we probably shouldn't bother:

<img width="419" alt="Screen Shot 2020-07-15 at 3 04 00 pm" src="https://user-images.githubusercontent.com/1754873/87505665-c16c5b80-c6ac-11ea-91de-0d777aebb600.png">